packages/manager/apps/hub/src/dashboard/dashboard.html

```diff
@@ -14,6 +14,29 @@
                 class="w-100"
                 tracking-prefix="{{ $ctrl.prefix }}"
             ></ovh-manager-hub-carousel>
+
+            <!-- This banner is ephemeral for domains repricing MANAGER-9696 -->
+            <oui-message
+                class="w-100"
+                data-ng-if="$ctrl.hasDomainRepricing"
+                data-type="info"
+            >
+                <strong
+                    data-translate="manager_hub_domain_repricing_part_1"
+                ></strong>
+                <br />
+                <span
+                    data-translate="manager_hub_domain_repricing_part_2"
+                ></span>
+                <br />
+                <span
+                    data-translate="manager_hub_domain_repricing_part_3"
+                ></span>
+                <br />
+                <span
+                    data-translate="manager_hub_domain_repricing_part_4"
+                ></span>
```

Comment:
**suggestion:** Instead of using `<span>`, `<p>` tag can be used. We can void using `<br />`. `<p>` may require addition of `mb-0` class.

Reply:
It's applied.